components/global/partial.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div>
+    <nuxt-content :document="content" />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    url: {
+      type: String,
+      required: true,
+    }
+  },
+  data: () => ({
+    content: []
+  }),
+  async fetch() {
+    this.content = await this.$content(this.url).fetch()
```

Comment:
The version we are using let's us use the composition api? or only the asyncData() method for example? If yes, I would suggest cleaning the code with it.

Reply:
Not without significant updating and setup. The initial partial examples I followed used `asyncData` but unfortunately they didn't work in this version. Ultimately decided not to update because this `docs` app is temporary, Marcelo is already working on the replacement.

Reply:
Alright.